content/blog/intro-to-web-accessability/index.md

```diff
@@ -0,0 +1,301 @@
+		<td><img src="./arm_injury.png" style="height: 200px" alt=""/><br/>Arm injury</td>
+		<td><img src="./new_parent.png" style="height: 200px" alt=""/><br/>New parent</td>
+	</tr>
+	<tr>
+		<th scope="row">See</th>
+		<td><img src="./blind.png" style="height: 200px" alt=""/><br/>Blind</td>
+		<td><img src="./cataract.png" style="height: 200px" alt=""/><br/>Cataract</td>
+		<td><img src="./distracted_driver.png" style="height: 200px" alt=""/><br/>Distracted driver</td>
+	</tr>
+	<tr>
+		<th scope="row">Hear</th>
+		<td><img src="./deaf.png" style="height: 200px" alt=""/><br/>Deaf</td>
+		<td><img src="./ear_infection.png" style="height: 200px" alt=""/><br/>Ear infection</td>
+		<td><img src="./bartender.png" style="height: 200px" alt=""/><br/>Bartender</td>
+	</tr>
+	<tr>
+		<th scope="row">Speak</th>
+		<td><img src="./non_verbal.png" style="height: 200px" alt=""/><br/>Non-verbal</td>
+		<td><img src="./laryngitis.png" style="height: 200px" alt=""/><br/>Laryngitis</td>
+		<td><img src="./heavy_accent.png" style="height: 200px" alt=""/><br/>Heavy accent</td>
+	</tr>
+</table>
+
+> Microsoft originally created this chart as part of their [Inclusive Toolkit](https://download.microsoft.com/download/b/0/d/b0d4bf87-09ce-4417-8f28-d60703d672ed/inclusive_toolkit_manual_final.pdf) manual
+
+Creating an application that's accessible means that you're making a better experience for _all_ of your users, ready for any context they might be using your app in.
+
+In addition to the moral and financial incentives (by opening the door to more users), many organizations have a legal requirement to meet accessibility. U.S. government software is subject to [Section 508](https://www.section508.gov/manage/laws-and-policies), which requires compliance to [the WCAG guidelines (which we'll touch on later)](#wcag). Likewise, private US companies may be subject to compliance due to the "Americans with Disabilities Act" (shortened to "ADA"). The U.S. isn't the only country with these requirements, either. According to [WCAG's reference page for various legal laws](https://www.w3.org/WAI/policies/), there are at least 40 such laws in place around the world.  
+
+> Please note that we are _not_ giving legal advice. This article is simply meant for educational purposes for individuals. Consult legal authorities for the appropriate jurisdiction
+
+
+Accessibility isn't a pure science, however. If you're sighted, this may be an abstract idea at first. However, think of it like this: the colors an app uses or a button's visual placement may convey different messages and meanings depending on their context. This same problem applies to users of screen-readers and other accessible tech as well, just with different constraints. Having the screen visually cluttered, thus making content difficult to read, might be a similar experience to having a site where the footer's contents are inexplicably read before the main page's content.
```

Comment:
```suggestion
Accessibility isn't a pure science, however. If you're sighted, this may be an abstract idea at first. However, think of it like this: the colors an app uses or a button's visual placement may convey different messages and meanings depending on their context. This same problem applies to users of screen-readers and other accessible tech as well, just with different constraints. If the screen is visually cluttered, the content may be more difficult to read. Likewise, different accessibility methods will lead to different experiences for users of assistive technology. In both of these scenarios, there may not be objectively correct answers - some may prefer a button placed visually to the left, while others might advocate for it on the right. Similarly, how something is read using a screen reader may make sense to some, but might be confusingly expressed to others.
```

Edited to explain _why_ I'm talking about this a bit more and clean up some of the phrasing